components/expressvars/package.json

```
@@ -3,6 +3,11 @@
   "version": "2.0.3",
   "description": "The Spectrum CSS vars for Express package",
   "license": "Apache-2.0",
+  "main": "dist/spectrum-metadata.json",
+  "scripts": {
+    "update": "gulp update",
```

```suggestion
    "update": "gulp update",
```

Since gulp is removed, the update work needs to be run from it's own script.

---

Done